bioptim/interfaces/acados_interface.py

```diff
@@ -679,6 +681,7 @@ def get_optimized_value(self) -> Union[list, dict]:
             "time_tot": self.ocp_solver.get_stats("time_tot")[0],
             "iter": self.ocp_solver.get_stats("sqp_iter")[0],
             "status": self.status,
+            "total_time_to_optimize": self.total_time_to_optimize,
```

Comment:
Is there a reason to create a third time holder? There is already `time_tot` and `real_time_to_optimize`. If it is important please add `total_time_to_optimize` to Ipopt as well as they are supposed to return (almost) exact same output structure

Reply:
Time for Acados and Ipopt were harmonized